model_analyzer/record/metrics_manager.py

```diff
@@ -563,9 +563,11 @@ def _print_run_config_info(self, run_config):
         collect_cpu_metrics_expect = cpu_only or len(self._gpus) == 0
         collect_cpu_metrics_actual = len(self._cpu_metrics) > 0
         if collect_cpu_metrics_expect and not collect_cpu_metrics_actual:
-            logger.info(
-                "CPU metric(s) are not being collected, while this profiling will run on CPU(s)."
-            )
+            if not self._cpu_warning_printed:
+                self._cpu_warning_printed = True
+                logger.warning(
+                    "CPU metric(s) are not being collected, while this profiling will run on CPU(s)."
```

Comment:
Is this proper English? Seems like some words are transposed in the warning message?

Reply:
reworded